src/main/java/frc/robot/subsystems/launcher.java

```diff
@@ -28,14 +30,18 @@ public launcher() {
     rightIndexSparkMax.setIdleMode(IdleMode.kCoast);
     rightIndexSparkMax.setInverted(true);
     rightIndexSparkMax.setSmartCurrentLimit(40);
+
     leftIndexSparkmax.follow(rightIndexSparkMax, true);
     leftIndexSparkmax.setIdleMode(IdleMode.kCoast);
+    leftIndexSparkmax.setSmartCurrentLimit(40);
 
     rightLauncherSparkMax.setIdleMode(IdleMode.kCoast);
     rightLauncherSparkMax.setInverted(true);
     rightLauncherSparkMax.setSmartCurrentLimit(40);
+
     leftLauncherSparkMax.setIdleMode(IdleMode.kCoast);
     leftLauncherSparkMax.follow(rightLauncherSparkMax, true);
+    leftLauncherSparkMax.setSmartCurrentLimit(40);
 
     pidController = leftLauncherSparkMax.getPIDController();
```

Comment:
pidController is overwritten on the next line so this likely isn't needed.

Reply:
Lines 46 and 47

Reply:
I don't understand how its overwritten, could you explain that? the reason I ask is because wouldn't the right need to overwritten as well since its the line on 46 defining it? I'm just making sure I understand this before I make any changes.